src/main/java/core/basesyntax/Ball.java

```diff
@@ -0,0 +1,22 @@
+package core.basesyntax;
+
+public class Ball {
+    protected final int randomBalls = 3;
+    private Colors colors;
+    private int number;
+
+    @Override
+    public String toString() {
+        return "Ball{" + "color='" + colors
```

Comment:
[Use name() for getting String representation of enum constants](https://mate-academy.github.io/jv-program-common-mistakes/java-core/oop/lottery#use-name-for-getting-string-representation-of-enum-constants)